gapic/src/main/com/google/gapid/perfetto/views/SelectionView.java

```diff
@@ -49,12 +49,14 @@ public void onDataChanged() {
   }
 
   @Override
-  public void onSelectionChanged(Selection selection) {
+  public void onSelectionChanged(Selection.MultiSelection selection) {
     for (Control c : group.getChildren()) {
       c.dispose();
     }
 
-    if (selection != null) {
+    if (selection == null) {
+      new Composite(group, SWT.NONE).requestLayout();
+    } else {
```

Comment:
why create an empty composite and ask for layout?

Reply:
Sorry, was trying to clear the info box at a de-selection, but didn't notice the dispose() code above already does the clearing duty. Have reverted it back in the new commit.